ui/components/institutional/interactive-replacement-token-notification/interactive-replacement-token-notification.tsx

```diff
@@ -93,8 +100,16 @@ const InteractiveReplacementTokenNotification = ({ isVisible }) => {
     };
 
     handleShowNotification();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [address, interactiveReplacementToken.oldRefreshToken, isUnlocked]);
+  }, [
+    address,
+    interactiveReplacementToken.oldRefreshToken,
+    isUnlocked,
+    dispatch,
+    keyring.type,
+    interactiveReplacementToken,
+    mmiActions,
+    address,
```

Comment:
`address` is listed twice and one can be removed

Reply:
Good catch!